packages/email-plugin/src/plugin.ts

```diff
@@ -80,6 +85,26 @@ import {
  *   ],
  * };
  * ```
+ * ### Channel aware SMTP settings
+ * If you would like to have different SMTP settings per channel, then you can configure the `transport` field to fetch the values.
+ * ```
+ * EmailPlugin.init({
+ *  transport: async (ctx) => {
+ *      // Get your SMTP settings based on the requestContext
+ *      const settings = await myService.getSmtpSettings(ctx);
```

Comment:
Is this example needed given the "Dynamic transport settings" section below? Maybe we can add this particular use-case to that section.